modules/tls-profiles-kubernetes-configuring.adoc

```diff
@@ -0,0 +1,117 @@
+// Module included in the following assemblies:
+//
+// * security/tls-profiles.adoc
+
+[id="tls-profiles-kubernetes-configuring_{context}"]
+= Configuring the TLS security profile for the control plane
+
+To configure a TLS security profile for the control plane, edit the `APIServer` custom resource (CR) to specify a predefined or custom TLS security profile. Setting the TLS security profile in the `APIServer` CR propagates the setting to the following control plane components:
+
+* Kubernetes API server
+* OpenShift API server
+* OpenShift OAuth API server
+* OpenShift OAuth server
+
+If a TLS security profile is not configured, the default TLS security profile is `Intermediate`.
+
+[NOTE]
+====
+The default TLS security profile for the Ingress Controller is based on the TLS security profile set for the API server.
+====
+
+.Sample `APIServer` CR that configures the `Old` TLS security profile
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: APIServer
+ ...
+spec:
+  tlsSecurityProfile:
+    old: {}
+    type: Old
+ ...
+----
+
+The TLS security profile defines the minimum TLS version and the TLS ciphers required to communicate with the control plane components.
```

Comment:
To confirm  the minimum TLS version and the TLS ciphers, we can run the following command line:
```
oc get openshiftapiservers.operator.openshift.io cluster -o json |jq .spec.observedConfig.servingInfo
{
  "cipherSuites": [
    "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256",
    "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256",
    "TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384",
    "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384",
    "TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256",
    "TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256",
    "TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256",
    "TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256",
    "TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA",
    "TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA",
    "TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA",
    "TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA",
    "TLS_RSA_WITH_AES_128_GCM_SHA256",
    "TLS_RSA_WITH_AES_256_GCM_SHA384",
    "TLS_RSA_WITH_AES_128_CBC_SHA256",
    "TLS_RSA_WITH_AES_128_CBC_SHA",
    "TLS_RSA_WITH_AES_256_CBC_SHA",
    "TLS_RSA_WITH_3DES_EDE_CBC_SHA"
  ],
  "minTLSVersion": "VersionTLS10"
}
```

Reply:
@wangke19 we have this section [1] that talks about getting details for the different profiles and their minimum TLS versions/ciphers, etc. 

Can you explain how what you're stating here is different than that (or just more detail about what you're suggesting we add)?

Also, same thing as the other comment. If we do want to make an update, we'll want to add it for all versions (this PR is just backporting what we added for 4.8 back to 4.7 and older). So we'd do any other enhancements in a separate PR.

[1] https://deploy-preview-34810--osdocs.netlify.app/openshift-enterprise/latest/security/tls-security-profiles.html#tls-profiles-view-details_tls-security-profiles

Reply:
The main purpose is to make sure that the changes we configured the `Old` TLS security profile are in line with our expected results. We are unable to get the detail from `oc describe apiserver cluster` for `Old` TLS security profile.

Reply:
@bergerhoffer 
>Also, same thing as the other comment. If we do want to make an update, we'll want to add it for all versions (this PR is just backporting what we added for 4.8 back to 4.7 and older). So we'd do any other enhancements in a separate PR.

Agree with you, we can supplement this part to 4.8 in a separate PR.